manifest/net.headassbtw/OpenXREyeTracking/info.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "OpenXREyeTracking",
+	"id": "net.headassbtw.OpenXREyeTracking",
+	"description": "Adds support for OpenXR-based eye trackers",
+	"category": "Hardware Integration",
+	"sourceLocation": "https://github.com/headassbtw/ResoniteOpenXREyeTracking",
+	"versions": {
+		"1.0.2": {
+			"releaseUrl": "https://github.com/headassbtw/ResoniteOpenXREyeTracking/releases/tag/1.0.2",
+			"artifacts": [
+				{
+					"url": "https://github.com/headassbtw/ResoniteOpenXREyeTracking/releases/download/1.0.2/OpenXREyeTracking.dll",
```

Comment:
In your RunPostInit when registering, the 2 ResoniteMod.Msg calls for the exceptions should be ResoniteMod.Error.

RegisterInputs also uses UniLog.Log but should use `ResoniteMod.Msg` instead, same in OpenXRInstance. (You may just need to add a new using)